modules/applications-create-using-cli-modify.adoc

```diff
@@ -0,0 +1,217 @@
+// Module included in the following assemblies:
+//
+// * applications/creating-new-applications.adoc
+
+[id="applications-create-using-cli-modify-{context}"]
+= Modifying application creation
+
+The `new-app` command generates {product-title} objects that will build, deploy,
+and run the application being created. Normally, these objects are created in
+the current project using names derived from the input source repositories or
+the input images. However, `new-app` allows you to modify this behavior.
+
+The set of objects created by `new-app` depends on the artifacts passed as
+input: source repositories, images, or templates.
+
+.`new-app` Output Objects
```

Comment:
If you were keeping this as a procedure module, you should put the reference table in a separate module.

The heading should be sentence case.